bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooargset.py

```diff
@@ -17,6 +17,20 @@
 
 
 class RooArgSet(RooAbsCollection):
+    def __init__(self, *args, **kwargs):
+        """Pythonization of RooArgSet constructor to support implicit
+        conversion from Python sets.
+        """
+        # Note: This simple Pythonization caused me days of headache.
```

Comment:
Nice! So in the end we can get rid of the constructor in C++ accepting a PyObject :+1:

Reply:
Yay! :D